README.md

```diff
@@ -42,9 +42,9 @@ power their ML infra, and Runhouse makes its best-in-class features accessible t
 ## 🤨 Who is this for?
 
 * 👩‍🔧 **Engineers, Researchers and Data Scientists** who don't want to spend 3-6 months translating and packaging
-their work to share it, and want to be able to iterate and improve production services, pipelines, and data artifacts
+their work to share it, and want to be able to iterate and improve their experiments, production services, pipelines, and data artifacts
```

Comment:
Good addition, but I would move experiments after pipelines, and wouldn't use "their" (I want to evoke that you can pick up any service, pipeline, etc. and iterate on it, not only your own).

Reply:
Done